explanation/custom-images.md

```diff
@@ -0,0 +1,8 @@
+Building custom images of Anbox Cloud based on AAOS is possible but may get tricky.
```

Comment:
How about saying that it is not something we recommend?

Reply:
Also, do we want to make it clear to the end user that it's a **system** image?

Reply:
We would need to explain what's a system image then, I think it's better not to go down that rabbit hole.